zarnitsa/DataAugmenterNLP.py

```diff
@@ -110,70 +112,79 @@ def _check_synset(self, name):
             my_file = os.path.join(
                 self.__class_local_path, "internal_data", "ppdb-2.0-tldr"
             )  # todo: defined but not used
-            wget.download(
-                "http://nlpgrid.seas.upenn.edu/PPDB/eng/ppdb-2.0-tldr.gz"
-            )
+            wget.download("http://nlpgrid.seas.upenn.edu/PPDB/eng/ppdb-2.0-tldr.gz")
         else:
-            raise KeyError(
-                f"Synset {name} is unknown! Load manually or fix the name"
-            )
+            raise KeyError(f"Synset {name} is unknown! Load manually or fix the name")
 
-    def augment_column_wordnet(self, text: str) -> str:
+    def augment_wordnet(self, text: str) -> str:
         """Augment column data using wordnet synset."""
         if not self.__aug_wdnt:
             print("Load WordNet synset")
             self.__aug_wdnt = naw.SynonymAug(aug_src="wordnet")
         text = self.__aug_wdnt.augment(text)
         return text
 
-    def augment_column_ppdb(self, text: str) -> str:
+    def augment_ppdb(self, text: str) -> str:
         """Augment column data using ppdb synset."""
         if not self.__aug_ppdb:
             print("Load PPDB synset")
             self._check_synset("ppdb")
             self.__aug_ppdb = naw.SynonymAug(
                 aug_src="ppdb",
-                model_path=self.__class_local_path
-                + "/internal_data/ppdb-2.0-tldr",
+                model_path=self.__class_local_path + "/internal_data/ppdb-2.0-tldr",
```

Comment:
I think we should consider usage of some libraries like Pathlib to operate with such things

Reply:
get it